application-task-default/src/main/resources/templates/notification/com.xwiki.task.internal.notifications.TaskChangedEvent.vm

```diff
@@ -0,0 +1,130 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+
+#template('notification/macros.vm')
+
+#set ($mainIcon = 'list')
+#set ($smallIcon = '')
+
+
+#macro (getGenericFieldDescription $customEventInfo)
+  ## The notification text will be available in `$eventText` after calling this macro.
```

Comment:
```suggestion
#macro (getGenericFieldDescription $customEventInfo $eventText)
```
Using the last param as the `return` value is usually used. I don't think it isn't added somewhere as the standard usage, but it's the actual practice
Same for macros below